src/components/Genre.tsx

```diff
@@ -0,0 +1,20 @@
+interface Props {
+  genres: string[];
+  selectedGenre: string;
+  onSelect: (genre: string) => void;
+}
+export default function Genre({ genres, selectedGenre, onSelect }: Props) {
+  return (
+    <div>
```

Comment:
This is GenreSelect (!) component on the design prototype. Even if we can imlement the group of buttons as s `<div>`, but, from the sematic and functional perpective it looks like `<nav>`
 
![Screenshot 2024-02-21 201443](https://github.com/keber23/react-learn/assets/79011727/a7a24fac-de95-49c6-a586-c831354c1912)